src/files/infrastructure/FileJSDataverseRepository.ts

```diff
@@ -36,13 +76,16 @@ export class FileJSDataverseRepository implements FileRepository {
       }, 1000)
     })
   }
-  // eslint-disable-next-line unused-imports/no-unused-vars
-  getFileUserPermissionsById(id: string): Promise<FileUserPermissions> {
-    // TODO - implement using js-dataverse
-    return new Promise((resolve) => {
-      setTimeout(() => {
-        resolve(FileUserPermissionsMother.create())
-      }, 1000)
-    })
+
+  getFileUserPermissionsById(id: number): Promise<FileUserPermissions> {
+    return getFileUserPermissions
+      .execute(id)
+      .then((jsFileUserPermissions) =>
+        JSFileMapper.toFileUserPermissions(id, jsFileUserPermissions)
+      )
+      .catch((error: WriteError) => {
+        console.error('Error getting the file user permissions from Dataverse', error)
```

Comment:
In other calls to js-dataverse I have not seen error logging. Do we want to keep this?

Reply:
removed!